languages/co-authors-plus-he_IL.po

```diff
@@ -0,0 +1,593 @@
+# Copyright (C) 2014 Co-Authors Plus
+# This file is distributed under the same license as the Co-Authors Plus package.
+msgid ""
+msgstr ""
+"Project-Id-Version: Co-Authors Plus 3.1-beta\n"
+"Report-Msgid-Bugs-To: http://wordpress.org/tag/co-authors-plus\n"
+"POT-Creation-Date: 2014-03-17 15:59:18+00:00\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"PO-Revision-Date: 2019-01-07 16:19+0200\n"
+"Language-Team: \n"
+"X-Generator: Poedit 2.1.1\n"
+"Last-Translator: \n"
+"Plural-Forms: nplurals=4; plural=(n==1 ? 0 : n==2 ? 1 : n>10 && n%10==0 ? "
+"2 : 3);\n"
+"Language: he_IL\n"
+
+#: co-authors-plus.php:300 co-authors-plus.php:401 co-authors-plus.php:487
+#: co-authors-plus.php:1290
+msgid "Authors"
+msgstr "מחברים"
+
+#: co-authors-plus.php:362
+msgid ""
+"<strong>Note:</strong> To edit post authors, please enable javascript or use "
+"a javascript-capable browser"
+msgstr ""
+"<strong>שימו לב:</strong> כדי לערוך את מחברי הפוסט יש לאפשר שימוש "
+"בJavascript בדפדפן."
```

Comment:
בJavascript בדפדפן should be Javascript בדפדפן

Reply:
Thanks for checking the file, @yaelirub. 🙌 I've just applied the change to the translation file.

Reply:
@nielslange , just making sure, is that kind of review you are looking for?